enhancements/etcd/automated-backups.md

```diff
@@ -0,0 +1,284 @@
+---
+title: automated-backups-of-etcd
+authors:
+  - "@hasbro17"
+reviewers:
+  - "@deads2k"
+  - "@dusk125"
+  - "@Elbehery"
+  - "@tjungblu"
+  - "@williamcaban"
+approvers:
+  - "@deads2k"
+  - "@williamcaban"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-03-17
+last-updated: 2023-03-17
+tracking-link:
+  - https://issues.redhat.com/browse/ETCD-81
+see-also:
+  - "https://issues.redhat.com/browse/OCPBU-252"
+  - "https://issues.redhat.com/browse/OCPBU-160"
+---
+
+
+# Automated Backups of etcd
+
+## Summary
+
+Enable the automated backups of etcd snapshots and other metadata necessary to restore an openshift cluster from a quorum loss scenario.
+
+## Motivation
+
+The [current documented procedure](https://docs.openshift.com/container-platform/4.12/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html) for performing a backup of an OpenShift
+cluster is manually initiated. This reduces the likelihood of a timely backup being available during a disaster recovery scenario.
+
+The procedure also requires gaining a root shell on a control plane
+node. Shell access to OpenShift control plane nodes access is generally
+discouraged due to the potential for affecting the reliability of the
+node.
+
+### Goals
+
+- One-time cluster backup can be triggered without requiring a root shell
+- Scheduled backups can be configured after cluster installation
+- Backups are saved locally on the host filesystem of control-plane nodes
+- This feature is validated with an e2e restore test that ensures the backups saved can be used to recover the cluster from a quorum loss scenario
+
+
+### Non-Goals
+- Have automated backups enabled by default with cluster installation
+- Save cluster backups to PersistentVolumes or cloud storage
+  - This would be a future enhancement or extension to the API
+- Automate cluster restoration
```

Comment:
Is this expected as a future enhancement?  Not stated here as in the prior bullet.